Wikipedia/View Controllers/WebView/WebViewController.m

```diff
@@ -2045,4 +2053,142 @@ - (void)promptZeroOff {
     [dialog show];
 }
 
+#pragma mark Native section title label overlays
```

Comment:
Pretty much all of this code can be moved into the new control class that handles the management of the section headers